src/NzbDrone.Core/Download/Clients/rTorrent/RTorrentProxy.cs

```diff
@@ -194,7 +194,7 @@ private XDocument ExecuteRequest(RTorrentSettings settings, string methodName, p
 
             if (!settings.Username.IsNullOrWhiteSpace())
             {
-                requestBuilder.NetworkCredential = new NetworkCredential(settings.Username, settings.Password);
+                requestBuilder.NetworkCredential = new BasicNetworkCredential(settings.Username, settings.Password);
```

Comment:
This one should be left as NetworkCredential so that rtorrent can use digest auth please

Reply:
Pushed - as discussed we need to put it into Readarr